webpack/set-webpack-public-path-plugin/src/SetPublicPathPlugin.ts

```diff
@@ -51,6 +51,11 @@ export interface ISetWebpackPublicPathOptions {
    * See the README for more information.
    */
   getPostProcessScript?: (varName: string) => string;
+
+  /**
+   * If true, find the last script matching the regexVariable (if it is set). If false, find the first matching script.
+   */
+  findLast?: boolean;
```

Comment:
What is the use case for this?

Reply:
In case there are multiple scripts that match the regex (usually 2, for some debugging purpose) and you need to use the 2nd one instead of the first.